xml/decoders/Zimo_MX820.xml

```diff
@@ -0,0 +1,324 @@
+
+    <variables>
+
+    <variable CV="1" comment="Short address" item="Short Address" default="03">
+      <shortAddressVal/>
+      <label>Primary Address</label>
+    </variable>
+
+<!--                                                                        -->
+<!-- DO NOT CHANGE THE VALUE OF CV29                                        -->
+<!--                                                                        -->
+    <variable item="CV29" CV="29" readOnly="yes" default="136">
+      <decVal min="136" max="136" />
+      <label>CV29</label>
+    </variable>
+
+      <variable item="Power On Time (cv33=0)" CV="3" comment="Power On Time (cv33=0)" default="01">
+        <decVal min="0" max="255"/>
+        <label>Power On Time (cv33=0)</label>
+      </variable>
+
+      <variable item="Power On Time (cv33=1)" CV="4" comment="Power On Time (cv33=1)" default="01">
+        <decVal min="0" max="255"/>
+        <label>Power On Time (cv33=1)</label>
+      </variable>
+
+      <variable item="Power On Time (cv33=2)" CV="5" comment="Power On Time (cv33=2)" default="01">
+        <decVal min="0" max="255"/>
+        <label>Power On Time (cv33=2)</label>
+      </variable>
+
+      <variable item="Power On Time (cv33=3)" CV="6" comment="Power On Time (cv33=3)" default="01">
+        <decVal min="0" max="255"/>
+        <label>Power On Time (cv33=3)</label>
+      </variable>
+
+      <variable CV="7" readOnly="yes" item="Decoder Version" >
+        <decVal min="0" max="255"/>
+        <label>Decoder Version</label>
+      </variable>
+
+      <variable item="Manufacturer" CV="8" readOnly="yes" default="165">
+        <decVal min="0" max="255"/>
+        <label>Manufacturer</label>
+      </variable>
+
+      <variable item="Decoder ID" CV="250" readOnly="yes" default="165">
+        <decVal min="0" max="255"/>
+        <label>Decoder ID</label>
+      </variable>
+<!--                                                                        -->
+<!-- Zimo accessory address is 9 bits. Low 6 bts in cv1, high 3 bits in cv9 -->
+<!--                                                                        -->
+      <variable item="Zimo Address" CV="1,9" mask="XXVVVVVV XXXXXVVV" comment="Zimo Address">
+        <splitVal/>
+        <label>Zimo Address</label>
+      </variable>
+
+<!--                                                                        -->
+<!-- NMRA address adds 3 bits to the lower end of the Zimo accessory address -->
+<!-- the Zimo subaddress from cv33                                          -->
+<!--                                                                        -->
+      <variable item="NMRA Address" CV="33,1,9" mask="XXXXXXVV XXVVVVVV XXXXXVVV" comment="NMRA Address">
+        <splitVal offset="-3"/>
+        <label>Zimo Address</label>
+      </variable>
+
+<!--                                                                        -->
+<!-- Definitions for cvs 1, 9 and 33 MUST come AFTER the splitvals (abve)   -->
```

Comment:
No need to change, but "abve" is misspelled. Should be "above". There are one more case of this misspelling in the file.